src/packagedcode/models.py

```diff
@@ -1064,6 +1080,17 @@ def is_datafile(cls, location, filetypes=tuple(), _bare_filename=False):
                     actual_type = T.filetype_file.lower()
                     return any(ft in actual_type for ft in filetypes)
 
+    @classmethod
```

Comment:
Use a plain attribute instead.... this is never computed.

Reply:
Updated, thanks!